yaq_traits/has-transformed-position.toml

```diff
@@ -0,0 +1,73 @@
+trait = "has-transformed-position"
+doc = """
+"""
+
+
+[state]
+
+[state.native_reference_position]
+type = "double"
+default = 0.0
+
+
+[messages]
+
+[messages.to_transformed]
```

Comment:
My understanding is that we had decided _not_ to make these exposed methods.

If we do keep them we _could_ make it possible to pass an ndarray in, though not sure we _should_.

Reply:
Yeah, I went rogue on that one...I imagine this has a utility as a debugging tool--similar to the `direct_serial_write` method of `uses-serial`.  I'm fine to change back if we think it's a mistake.

Interesting idea re: ndarray type.  I share your uncertainty about implementing such a type, which I guess means we don't do it for now...

Reply:
I'm okay with it, I think, I was the one advocating _for_ these methods... And I'm completely fine with saying no to ndarray until and unless it is shown to be needed.

Reply:
I'm :-1: on these methods but happily overruled by you @ddkohler since you are actually doing the work :smile:

Reply:
majority rules; will include function in spec for now